scheduled-jobs/build/poll-payload/Jenkinsfile

```diff
@@ -58,6 +59,36 @@ def publishRPMs(String releaseStream, Map latestRelease, Map previousRelease) {
     )
 }
 
+def startBuildMicroshiftJob(String releaseStream, Map latestRelease, Map previousRelease) {
+    // This function is invoked when a new x86_64 is created and accepted by the Release Controller
+    def buildVersion = commonlib.extractMajorMinorVersion(releaseStream)
+    def x86_64_nightly = latestRelease.name
+    // Get all nightlies matching the x86_64 nightly
+    echo "Get nightlies matching the x86_64 nightly ${x86_64_nightly}..."
+    def cmd = "doozer --assembly=stream --arches x86_64,aarch64 --group openshift-${buildVersion} --working-dir ./doozer-working get-nightlies --matching ${x86_64_nightly}"
+    def res = commonlib.shell(script: cmd, returnAll: true)
+    if (res.returnStatus != 0) {
+        if (res.combined.contains("Found no nightlies") || res.combined.contains("No sets of equivalent nightlies")) {
+            // Couldn't find consistent and accepted nightlies matching the x86_64 nightly
+            echo "WARNING: Couldn't find consistent and accepted nightlies matching the x86_64 nightly ${x86_64_nightly}. Will not trigger build-microshift."
+            return
+        }
+        error("Error getting nightlies matching ${x86_64_nightly}: ${res.stderr}")
+    }
+    // Trigger build-microshift
+    def nightlies = res.stdout.trim()
+    build(
+        job: "../aos-cd-builds/build%2Fbuild-microshift",
+        propagate: false,
+        parameters: [
+            string(name: 'BUILD_VERSION', value: buildVersion),
+            string(name: 'ASSEMBLY', value: "test"),
```

Comment:
Is there a reason this is "test" instead of "stream"?

Reply:
Builds for nightlies are just for testing purpose and should never ship via Errata. Using test assembly is to prevent from being swept by Elliott.

Reply:
I thought so, thanks for the confirmation :+1: